routers/web/repo/issue_pin.go

```diff
@@ -33,22 +36,25 @@ func IssuePinOrUnpin(ctx *context.Context) {
 
 // IssueUnpin unpins a Issue
 func IssueUnpin(ctx *context.Context) {
-	issue, err := issues_model.GetIssueByIndex(ctx.Repo.Repository.ID, ctx.ParamsInt64(":id"))
+	issue, err := issues_model.GetIssueByID(ctx, ctx.ParamsInt64(":id"))
```

Comment:
Why not use `index`? I think almost all of other places are using index for issues/pulls.